besu/src/test/java/org/hyperledger/besu/cli/custom/RpcAuthFileValidatorTest.java

```diff
@@ -27,11 +27,16 @@
 @RunWith(MockitoJUnitRunner.StrictStubs.class)
 public class RpcAuthFileValidatorTest {
 
-  private static final String CORRECT_TOML = "/auth_correct.toml";
-  private static final String DUPLICATE_USER_TOML = "/auth_duplicate_user.toml";
-  private static final String INVALID_TOML = "/auth_invalid.toml";
-  private static final String INVALID_VALUE_TOML = "/auth_invalid_value.toml";
-  private static final String NO_PASSWORD_TOML = "/auth_no_password.toml";
+  private static final String CORRECT_TOML = "/rpcauth/auth_correct.toml";
+  private static final String DUPLICATE_USER_TOML = "/rpcauth/auth_duplicate_user.toml";
+  private static final String INVALID_TOML = "/rpcauth/auth_invalid.toml";
+  private static final String INVALID_GROUPS_VALUE_TOML = "/rpcauth/auth_invalid_groups_value.toml";
```

Comment:
do we want to put in the groups etc. at this stage? Or just the PRIVACY_KEY?

Reply:
This is just testing the parsing of groups that already exists in Auth file parsing. It doesn't add groups or roles as claims.